keras_nlp/tokenizers/word_piece_tokenizer.py

```diff
@@ -150,7 +150,7 @@ class WordPieceTokenizer(tokenizer.Tokenizer):
             tensor and padded/trimmed so all outputs are of sequence_length.
         lowercase: bool, defaults to `True`. If true, the input text will be
```

Comment:
Maybe we should do `lowercase=False` by default too. As we start looking at multi-lingual, that seems like the least surprising thing here.

Split true by default is important for usability, but otherwise we don't modify the input by default.